e2e/tests/api_tests/settle-discontinue/api_settle_discontinue_claim_2v1_spec_test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+
+const config = require('../../../config.js');
+
+Feature('CCD Settle and discontinue claim 2v1 API test @api-spec');
+Scenario('Settle claim 2v1 scenario', async ({I, api_spec}) => {
+  await api_spec.createClaimWithRepresentedRespondent(config.applicantSolicitorUser, 'TWO_V_ONE');
+  await api_spec.createCaseFlags(config.hearingCenterAdminWithRegionId2);
+  await api_spec.manageCaseFlags(config.hearingCenterAdminWithRegionId2);
+  await api_spec.settleClaimSelectClaimant(config.applicantSolicitorUser, 'YES');
+});
+
+Scenario.only('Discontinue claim 2v1 scenario', async ({I, api_spec}) => {
```

Comment:
Is this Scenario Only neeeded?

Reply:
You are right deleted the Only from the test. My mistake, thanks @ShwetaTandel-hmcts